opengemini-client-api/src/main/java/io/opengemini/client/api/OpenGemin.java

```diff
@@ -0,0 +1,52 @@
+package io.opengemini.client.api;
+
+import io.opengemini.client.api.model.BatchPoints;
+import io.opengemini.client.api.model.Point;
+import io.opengemini.client.api.model.Query;
+import io.opengemini.client.api.model.QueryResult;
+import io.opengemini.client.api.model.config.Address;
+import io.opengemini.client.api.model.config.AuthConfig;
+import io.opengemini.client.api.model.config.BatchConfig;
+
+import java.util.List;
+
+public interface OpenGemin {
+
+    public void WriteBatchPoints(BatchPoints bps);
```

Comment:
first letter of method name should be in lower case

Reply:
Thank you. I will modify it.